### DISCUSSION TITLE
Right way to control sequence of related 'create's?

### DISCUSSION THREAD

**Discussion by pbelbin** [prisma/prisma#2776] [Q&A] [answered]
Hey there!

This is a TypeScript question I suppose:

If I am using `prisma.{entity}.create({data: { // list of properties }})` to create an entity, and the entity has multiple relation collections available, how do I control the order in which each of the related collections will be handled?

It seems that currently, because the relation collections are properties within the created entity that's at the root of the 'tree' or 'graph', that these are just being handled in alphabetical order.

Is there a way for me to control the order of these related graph executions?

If not, can we add this?  It would be really helpful to be able to control the order of execution.

**Comment by ryands17** [comment, marked as answer]
Hey @pbelbin :wave: 
Could you create a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) explaining the above use case?

**Comment by ludralph** [comment]
Hi @pbelbin,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
